assets/l10n/app_en.arb

```diff
@@ -27,18 +31,298 @@
   "@profileButtonSendDirectMessage": {
     "description": "Label for button in profile screen to navigate to DMs with the shown user."
   },
-  "cameraAccessDeniedTitle": "Permissions needed",
-  "@cameraAccessDeniedTitle": {
-    "description": "Title for dialog when the user needs to grant permissions for camera access."
+  "permissionsNeededError": "Error",
+  "@permissionsNeededError": {
+    "description": "General title for access dialog when we encounter an unknown error."
```

Comment:
This doesn't seem to get used.

Reply:
There were some unreplaced strings this was meant to stand in for, so this is being used now.